criu/net.c

```diff
@@ -3178,19 +3190,53 @@ static inline int nftables_network_unlock(void)
 #endif
 }
 
+static int iptables_has_criu_jump_target(void)
```

Comment:
This is a confusing name, it returns 0 on success, but in c zero is false, and from the name of the function it should imply that "iptables does not have criu jump target".

Reply:
Thank you Pavel! I have opened the following pull request to address this: https://github.com/checkpoint-restore/criu/pull/2337